test/rack_utils.go

```diff
@@ -145,19 +155,22 @@ func isNamespaceRackEnabled(
 	}
 
 	var pod asdbv1beta1.AerospikePodStatus
-	for _, p := range aeroCluster.Status.Pods {
-		pod = p
+	for podIndex := range aeroCluster.Status.Pods {
```

Comment:
```suggestion
	for podName := range aeroCluster.Status.Pods {
```

Please update this var wherever applicable in this PR as it is a map